Formula/gojq.rb

```diff
@@ -1,9 +1,11 @@
 class Gojq < Formula
   desc "Pure Go implementation of jq"
   homepage "https://github.com/itchyny/gojq"
-  url "https://github.com/itchyny/gojq/archive/v0.12.0.tar.gz"
-  sha256 "3f280536f53ab6670b89424b0cd295d096b2d2928a5ed5d53bc3a23333051c05"
+  url "https://github.com/itchyny/gojq.git",
+      tag:      "v0.12.0",
+      revision: "203d5e1dfd435460500ea5b00f71eca48bf1cb88"
   license "MIT"
```

Comment:
```suggestion
  license "MIT"
  revision 1
```
This will update `gojq` for everyone who has it installed when they do `brew upgrade.`